packages/stores/types.ts

```diff
@@ -293,6 +339,26 @@ type TTradersHubStore = {
     selected_account_type: string;
     no_CR_account: boolean;
     no_MF_account: boolean;
+    selected_region: string;
+    openFailedVerificationModal: (from_account: string) => void;
+    multipliers_account_status: string | null;
+};
+
+type TGtmStore = {
+    visitorId: string;
+    common_variables: string;
+    accountSwitcherListener: () => void;
+    pushDataLayer: (event: string) => void;
+    pushTransactionData: () => void;
+    eventHandler: () => void;
+    setLoginFlag: () => void;
+};
+
+type TRudderstackStore = {
+    identifyEvent: () => Promise<void>;
```

Comment:
@yaswanth-deriv It accepts a parameter, Please double check 🙏🏻

Reply:
Sure i will check and update it